src/app/components/top-menu/sequential-form/sequential-form.component.html

```diff
@@ -0,0 +1,44 @@
+<app-draggable-window [width]="'40%'" [height]="'65%'" [isResizeable]="true" [top]="'5%'" [right]="'2%'"
+                      (closeCallback)="this.close()" [title]="'Sequential Solver'">
+
+
+  <div [ngStyle]="{'visibility': this.showEdit ? 'visible' : 'hidden'}" class="matrix-input-content">
+    <textarea cols="40" rows="15" id="ef-text-input"
+                name="efTextInput" [(ngModel)]="this.efForm"></textarea>
+    <textarea cols="40" rows="15" id="ef-text-input2"
+                name="efTextInput" [(ngModel)]="this.solver.variableNames"></textarea>
+  </div>
+
+
+  <div></div>
+  <div class="dimensions-input" align="center">
+    <div>Equilibria: </div>
+    <label for="nash">Nash</label>
+    <input type="checkbox" id="nash" name="nash" [(ngModel)]="this.include_nash">
+    <label for="seq">Sequential</label>
+    <input type="checkbox" id="seq" name="seq" [(ngModel)]="this.include_sequential">
+  </div>
+  <div class="dimensions-input" align="center" >
+    <div>Restrictions: </div>
+    <label for="rs">Pure Strategies </label>
+    <input type="checkbox" id="rs" name="rs" [(ngModel)]="this.restrict_strategy">
+    <label for="rb">Pure Beliefs </label>
+    <input type="checkbox" id="rb" name="rb" [(ngModel)]="this.restrict_belief">
+  </div>
+  <div class="buttons-container" *ngIf="this.userActionController && this.userActionController.strategicForm">
+    <div class="button" (click)="this.postGameTree()">Solve</div>
+    <div class="button" (click)="this.createEfForm()">Refresh</div>
+    <div class="button" (click)="this.showEdit = !this.showEdit">Edit</div>
+  </div>
+
+
```

Comment:
emptylines should be removed. (or leave 1)

Reply:
We'll remove them.